android/quest/src/main/java/org/smartregister/fhircore/quest/ui/questionnaire/QuestionnaireViewModel.kt

```diff
@@ -489,8 +490,8 @@ constructor(
                 context = context,
                 transformSupportServices = transformSupportServices,
                 structureMapProvider = { structureMapUrl: String?, _: IWorkerContext ->
-                  structureMapUrl?.substringAfterLast("/")?.let {
-                    defaultRepository.loadResource(it)
+                  structureMapUrl?.extractLogicalIdUuid()?.let { structureMapId ->
```

Comment:
`extractLogicalIdUuid()` returning an empty string but previous extension `substringAfterLast("/")` working fine. We can revert this change